### ISSUE TITLE
Feature request: Option to only show street lighting quests at night

### ISSUE THREAD

**Issue by mretoolforge** [streetcomplete/StreetComplete#3925]
**Use case**

Often, eyeballing street light poles during daylight hours isn't a great way to answer questions such as "Is this stretch of way lit?" or "Is this bus stop lit?" If you choose not to answer these during the day, these quests contribute to visual clutter, and while turning them on and off or using a preset is an option it is a bit unwieldy since you would have to remember to switch it and the lighting questions are spread between multiple quests.

**Proposed Solution**
There could be a check/toggle box in the settings for this. I think this might be fairly straightforward to implement given that StreetComplete already switches the rendering theme based on day time or night time; this feature would do the same thing but for whether or not lighting prelates quests get shown.

**Comment by riQQ** [comment]
See also https://github.com/streetcomplete/StreetComplete/issues/3248#issuecomment-972375735.

**Comment by endolith** [comment]
Yeah, I cannot tell if a place is lit or not during the day, and these quests just clutter up the interface for no reason, so I've disabled them altogether.
